gnd/src/main/res/values/styles.xml

```diff
@@ -158,8 +158,38 @@
     <item name="android:layout_marginRight">@dimen/field_horizontal_margin</item>
   </style>
 
+  <!-- NavDrawer Header -->
   <style name="ShapeAppearanceOverlay.App.CornerSize50Percent" parent="">
     <item name="cornerSize">50%</item>
   </style>
 
+  <style name="Header">
```

Comment:
Naming nit: Since these styles are global, it might be helpful to give a more specific name to these styles, e.g. `NavDrawerHeader` or similar.